routes/api.php

```diff
@@ -83,5 +83,6 @@
         Route::get('/admin/nested-categories/{subcategory_id}', [CategoryController::class, 'getSubcategoryParentCategories']);
         Route::post('/admin/profile-edit', [ChangeNameEmailController::class, 'restore']);
         Route::patch('/admin/password-edit', [AdminController::class, 'updatePassword']);
+        Route::get('/admin/categories-data', [CategoryController::class, 'listOfCategories']);
```

Comment:
i think it is better for this to be /admin/categories. just update the other one to /admin/categories-data